gammapy/detect/test_statistics.py

```diff
@@ -196,7 +319,7 @@ def _ts_value(position, counts, exposure, background, kernel, flux,
         Background map.
     exposure : array
         Exposure map.
-    kernel : `astropy.convolution.Kernel2D`
+    kernel : astropy.convolution.core.Kernel2D
```

Comment:
I had merge conflicts and just copied the files, I guess that's were this "backward" changes come from. Thanks for the comments so far! I will address them soon and also add an example to gammapy-extra.